pkg/scanners/clairv4/convert.go

```diff
@@ -149,6 +152,15 @@ func vulnerabilities(vulnerabilities map[string]*claircore.Vulnerability, ids []
 	return vulns
 }
 
+func name(original string) string {
```

Comment:
Can we have a more descriptive function name, maybe:

```suggestion
func vulnName(original string) string {
```